armi/physics/fuelCycle/assemblyRotationAlgorithms.py

```diff
@@ -19,17 +19,23 @@
 These algorithms are defined in assemblyRotationAlgorithms.py, but they are used in:
 ``FuelHandler.outage()``.
 
-.. warning:: Nothing should do in this file, but rotation algorithms.
+.. warning:: Nothing should go in this file, but rotation algorithms.
 """
+import math
+
 from armi import runLog
 from armi.physics.fuelCycle.hexAssemblyFuelMgmtUtils import (
     getOptimalAssemblyOrientation,
 )
 from armi.physics.fuelCycle.settings import CONF_ASSEM_ROTATION_STATIONARY
 
 
+def _rotationNumberToRadians(rot: int) -> float:
+    return rot * math.pi / 3
```

Comment:
```suggestion
def _rotationNumberToRadians(rot: int) -> float:
    """Convert a rotation number to radians, assuming a HexAssembly."""
    return rot * math.pi / 3
```

Reply:
We have a lot of implicit assumptions of hex-based assemblies around ARMI. They are frequently traps. I just want to call them out.

You can certainly fix/change the wording, to your heart's content.